### PR TITLE
Added content for stacks and queues

### PR DESCRIPTION
**Pull request by spriyam**
Added stacks-queues.md page. Updated readme.md to add the stacks-queues page in it.

### CODE REVIEW

File: topics/stacks-queues.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- [Blog by Educative](https://www.educative.io/blog/data-structures-stack-queue-java-tutorial)
+- [Princeton University - Introduction to Stacks](https://introcs.cs.princeton.edu/java/43stack/) 
+
+## Problem Sets
+
+### Easy problems
+Problem Statement | Solution | WWC example
+--- | --- | ---
+[Valid Parentheses](https://leetcode.com/problems/valid-parentheses/) | [Solution link](https://leetcode.com/problems/valid-parentheses/solution/) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
+[Min Stack](https://leetcode.com/problems/min-stack/) | [Solution discussion](https://leetcode.com/problems/min-stack/solution/) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```
Comment (elaguerta) [inline topics/stacks-queues.md:22]: It looks like the solution for Min Stack is behind the paywall. I suggest directing to the solution discussion instead:
```suggestion
[Min Stack](https://leetcode.com/problems/min-stack/) | [Solution discussion](https://leetcode.com/problems/min-stack/discuss/?currentPage=1&orderBy=hot&query=) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```

File: topics/stacks-queues.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+### Medium problems
+Problem Statement | Solution | WWC example
+--- | --- | ---
+[Binary Tree zig zag level order tree traversal ](https://leetcode.com/problems/binary-tree-zigzag-level-order-traversal/) | [Solution discussion](https://leetcode.com/problems/binary-tree-zigzag-level-order-traversal/solution/) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```
Comment (elaguerta) [inline topics/stacks-queues.md:28]: Same thing here, solution behind the paywall. 
```suggestion
[Binary Tree zig zag level order tree traversal ](https://leetcode.com/problems/binary-tree-zigzag-level-order-traversal/) | [Solution discussion](https://leetcode.com/problems/binary-tree-zigzag-level-order-traversal/discuss/?currentPage=1&orderBy=hot&query=) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```

File: topics/stacks-queues.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+[Binary Tree inorder traversal](https://leetcode.com/problems/binary-tree-inorder-traversal/) | [Solution discussion](https://leetcode.com/problems/binary-tree-inorder-traversal/solution/) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```
Comment (elaguerta) [inline topics/stacks-queues.md:29]: I'm sure no one will notice except for me, but let's label the solution 'solution link' to let people know it's the Solution and not the Discussion :)

```suggestion
[Binary Tree inorder traversal](https://leetcode.com/problems/binary-tree-inorder-traversal/) | [Solution link](https://leetcode.com/problems/binary-tree-inorder-traversal/solution/) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```

File: topics/stacks-queues.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+[Basic Calculator - II](https://leetcode.com/problems/basic-calculator-ii/) | [Solution discussion](https://leetcode.com/problems/basic-calculator-ii/solution/) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```
Comment (elaguerta) [inline topics/stacks-queues.md:30]: Same thing here
```suggestion
[Basic Calculator - II](https://leetcode.com/problems/basic-calculator-ii/) | [Solution link](https://leetcode.com/problems/basic-calculator-ii/solution/) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```

File: topics/stacks-queues.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+
+### Hard problems
+Problem Statement | Solution | WWC example
+--- | --- | ---
+[Trapping rain water](https://leetcode.com/problems/trapping-rain-water/) | [Solution discussion](https://leetcode.com/problems/kth-smallest-element-in-a-sorted-matrix/discuss/?currentPage=1&orderBy=most_votes&query=) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```
Comment (elaguerta) [inline topics/stacks-queues.md:36]: This one is an error, Trapping Rain Water is missing the solution link
```suggestion
[Trapping rain water](https://leetcode.com/problems/trapping-rain-water/) | [Solution link](https://leetcode.com/problems/trapping-rain-water/solution/) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```

File: topics/stacks-queues.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+[Largest rectangle in histogram](https://leetcode.com/problems/largest-rectangle-in-histogram/) | [Solution discussion](https://leetcode.com/problems/wiggle-sort-ii/discuss/?currentPage=1&orderBy=most_votes&query=) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```
Comment (elaguerta) [inline topics/stacks-queues.md:37]: Same thing here, missing solution link
```suggestion
[Largest rectangle in histogram](https://leetcode.com/problems/largest-rectangle-in-histogram/) | [Solution discussion](https://leetcode.com/problems/largest-rectangle-in-histogram/discuss/?currentPage=1&orderBy=hot&query=) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/)
```

**Review by elaguerta — CHANGES_REQUESTED** [review]
Great job! I liked your resources selection - that CMU article is great.
Nice problem selection too, covers the important techniques and topics.
See suggestions in-line. The only errors I found were the last too problems were missing solution links (they had the wrong links, probably left over from the template). 
Once you have resolved the comments, please feel free to merge it in, it does not need a re-review from me. Thank you for this contribution!

**Comment by spriyam** [comment]
Incorporated all the review comments

**Pull request merged**